extension/content.js

```diff
@@ -170,6 +170,15 @@ function addPatchDiffLinks() {
 	`);
 }
 
+function removeDiffSigns() {
+	$('.blob-code-deletion .blob-code-inner, .blob-code-addition .blob-code-inner').each((index, element) => {
+		const $element = $(element);
+		const html = $element.html();
+
+		$element.html(html.replace(/^[+-]/, ' '));
```

Comment:
The first text node includes the indentation (eg "+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"), stripping the entire node would remove any indentation on diff lines (If I'm understanding you correctly). Should I change `html` to be the first text node, and still run the replace?